src/Nox.Generator/NoxFileGenerator.cs

```diff
@@ -0,0 +1,182 @@
+﻿using Nox.Generator.Common;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using YamlDotNet.Core;
+using YamlDotNet.Serialization.NamingConventions;
+using YamlDotNet.Serialization;
+using Nox.Solution;
+using System.Reflection;
+using Nox.Generator.Application.Commands;
+
+namespace Nox.Generator;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("MicrosoftCodeAnalysisCorrectness", "RS1035:Do not use APIs banned for analyzers", Justification = "<Pending>")]
+public class NoxFileGenerator
+{
+    private readonly List<string> _errors = new();
+    private readonly IEnumerable<string> _noxYamls = Array.Empty<string>();
+
+    public NoxFileGenerator(IEnumerable<string> noxYamls)
+    {
+#if DEBUG
+        if (!Debugger.IsAttached)
+        {
+            //Debugger.Launch();
+        }
+#endif
+        _noxYamls = noxYamls;
+    }
+
+    public void GenerateFiles()
+    {
+        try
+        {
+            if (TryGetGeneratorConfig(out var config) && TryGetNoxSolution(out var solution))
+            {
+                var codeGeneratorState = new NoxSolutionCodeGeneratorState(solution, Assembly.GetEntryAssembly()!);
+
+                var generatorFlows = new (NoxGeneratorKind GeneratorKind, bool IsEnabled)[]
+                {
+                    (NoxGeneratorKind.None,true),
+                    (NoxGeneratorKind.Domain,config.Domain),
+                    (NoxGeneratorKind.Infrastructure,config.Infrastructure),
+                    (NoxGeneratorKind.Presentation,config.Presentation),
+                    (NoxGeneratorKind.Application,config.Application),
+                    (NoxGeneratorKind.Ui,config.Ui)
+                }
+                .Where(x => x.IsEnabled)
+                .Select(x => x.GeneratorKind)
+                .ToArray();
+
+                //var generatorInstances = Assembly
```

Comment:
Assembly .GetExecutingAssembly().GetTypes() is not compatible with netstandard2.0 for any reasons. It needs to be changed.